src/DssSpell.sol

```diff
@@ -20,11 +20,7 @@ pragma experimental ABIEncoderV2;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
-import { VatAbstract, LerpFactoryAbstract, SpotAbstract} from "dss-interfaces/Interfaces.sol";
-
-interface LerpAbstract {
-    function tick() external returns (uint256);
-}
+import {VestAbstract} from "dss-interfaces/Interfaces.sol";
```

Comment:
I would recommend to import as follow to avoid fetching all the interfaces (this layout should not be used for the spells):
`import "dss-interfaces/dss/VestAbstract.sol";`

Reply:
Seems to generate the same code regardless of the import syntax. Maybe this is being pulled in elsewhere?

Reply:
Oh! yeah, you're right. I think could be related to using latest hevm 0.49.0 release.

Reply:
tried now with hevm 0.48.0 and I'm getting the same result.

Reply:
actually, double checking, I think you need to re-build (try without opts to speed the process for testing) and it should work.

Reply:
this seems to happen only with multiple import layout

Reply:
I'm pretty sure Naz is right, make sure after setting the new method you are rebuilding and only after trying to make the flat.